back/boxtribute_server/graph_ql/resolvers.py

```diff
@@ -332,18 +331,32 @@ def resolve_create_qr_code(_, info, box_label_identifier=None):
 
 @mutation.field("createBox")
 @convert_kwargs_to_snake_case
-def resolve_create_box(_, info, box_creation_input):
+def resolve_create_box(*_, box_creation_input):
```

Comment:
I like the `*_` hiding of unused arguments. Already on the list for refactoring :smile: